source/documentation/index.md

```diff
@@ -67,13 +67,21 @@ Use this API to:
 - Request the NOP Waiver authorisation status of 1-3000 EORIs passed as an array.
 - Run tests in the HMRC sandbox environment.
 
-## Getting started 
+## Developer Setup
 
-### Developer Setup
+In the context of this API, the term "developer" refers to software developers who either work directly for CSPs or are contracted to CSPs through third parties.
+
+### Getting started
+
+Developers must follow the steps below before you can use your software in the test and live environment and access this API:
+
+1. **[Register for a developer account](https://developer.service.hmrc.gov.uk/developer/registration)** on the HMRC Developer Hub. 
+2. To **create** a standard application, follow the instructions on [Using the Developer Hub](https://developer.qa.tax.service.gov.uk/api-documentation/docs/using-the-hub)
```

Comment:
Hi Yasmin,
Should this link not be this: https://developer.service.hmrc.gov.uk/api-documentation/docs/api instead of the QA one? Not 100% sure though

Reply:
Thanks Reece will change to https://developer.service.hmrc.gov.uk/api-documentation/docs/using-the-hub